source/windows/win_delay_load_hook.c

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
```

Comment:
Actually hold up.  We should make sure everything is kosher license-wise with this.